model/process/deprecated/registry-deprecated.yaml

```diff
@@ -25,3 +25,4 @@ groups:
         deprecated: 'Replaced by `process.executable.build_id.htlhash`'
         brief: >
           "Deprecated, use `process.executable.build_id.htlhash` instead."
+        examples: ['600DCAFE4A110000F2BF38C493F5FB92']
```

Comment:
are examples required now?

Reply:
they are (and have been required) except for enums